com.unity.formats.fbx/package.json

```diff
@@ -4,7 +4,8 @@
   "version": "2.0.3-preview.3",
   "dependencies": {
     "com.unity.timeline": "1.0.0",
-    "com.autodesk.fbx": "2.0.1-preview.1"
+    "com.autodesk.fbx": "2.0.1-preview.1",
+    "com.unity.recorder": "2.1.0-preview.1"
```

Comment:
can you add the line before timeline (to avoid a merge conflict when we bump to 3.0.0)